test/jb_directory/set_terminals_of.test.js

```diff
@@ -72,7 +73,7 @@ describe('JBDirectory::setTerminalsOf(...)', function () {
 
     await expect(jbDirectory.connect(projectOwner).setTerminalsOf(PROJECT_ID, terminals))
       .to.emit(jbDirectory, 'SetTerminals')
-      //.withArgs(PROJECT_ID, terminals, projectOwner.address); -->  AssertionError: expected undefined to equal (...) ?!
+    //.withArgs(PROJECT_ID, terminals, projectOwner.address); -->  AssertionError: expected undefined to equal (...) ?!
```

Comment:
Gave it another try, still don't know how to solve this (event is emitted, as tested, tho)

Reply:
can we get this with a forge test?

Reply:
fixed by removing `indexed` from terminals arg to the event